holoviews/plotting/bokeh/annotation.py

```diff
@@ -130,6 +130,8 @@ class LineAnnotationPlot(ElementPlot, AnnotationPlot):
 
     style_opts = line_properties + ['level']
 
+    apply_ranges = False
```

Comment:
I think this should probably be a boolean parameter.

Reply:
Ah, of course.